### DISCUSSION TITLE
Suggestion: Granaries & Warehouse on "Emptying" Orders

### DISCUSSION THREAD

**Discussion by arkimedes1967** [Keriew/augustus#253] [Ideas]
Please exclude this thread.

**Comment by cdsmith-umn** [comment]
This would be nice because it takes forever to empty these buildings when you want to do so.

**Comment by crudelios** [comment]
Isn't this now fixed with the option of sending more goods at once?

**Comment by crudelios** [comment]
Ok, thanks. I'll see if it makes sense to empty 4 goods at a time with the option enabled.

**Comment by crudelios** [comment]
I have no time for such an overhaul right now. One thing at a time,or nothing will get done!

**Comment by crudelios** [comment]
I'd rather have a per-resource "Empty" option, like in Pharaoh actually. Much more flexible!
